mbed_tools_ci_scripts/utils/filesystem_helpers.py

```diff
@@ -119,14 +119,17 @@ def __exit__(self, exc: Any, value: Any, tb: Any) -> None:
         """Context manager exit point."""
         self.cleanup()
 
+    def _cleanup_using_lib_facility(self) -> None:
+        try:
+            self._tmp_dir_context_manager.cleanup()
+        except FileNotFoundError as e:

```

Comment:
Is this necessary?

Reply:
yes. cleanup may be used multiple times (ensure idempotency)